wordpress-updatable/versions/0.1.1/schema.yaml

```diff
@@ -11,9 +11,6 @@ x-google-marketplace:
     - Feature
     recommended: true
 
-  managedUpdates:
-    kalmSupported: true
```

Comment:
Hm - each release is intended to be immutable, so we should avoid changing 0.1.1 and 0.1.2 (they won't be republished)